packages/framework-provider-local-infrastructure/test/scheduler.test.ts

```diff
@@ -0,0 +1,70 @@
+import { BoosterConfig } from '@boostercloud/framework-types'
+import { describe } from 'mocha'
+import { fake, replace, restore } from 'sinon'
+import { configureScheduler } from '../src/scheduler'
+import { expect } from '../test/expect'
+import * as scheduler from 'node-schedule'
+
+describe('scheduler', () => {
+  let mockConfig: any
+
+  const userProject = {
+    boosterTriggerScheduledCommand: () => {
+      console.log('triggering scheduled command...')
```

Comment:
I'd avoid having any console.log in the unit tests. I know there are some already, but I believe unit tests shouldn't log unless specified, and given that there is no logger module for the unit tests yet I'd skip it 😄

Reply:
Sure thing! I just wanted to have a "do-nothing" function as the `boosterTriggerScheduledCommand`. I can try instead something like `boosterTriggerScheduledCommand: () => {}`, would that be ok?

Reply:
Sure!